python/CHANGELOG.rst

```diff
@@ -2,8 +2,21 @@
 [0.6.4] - 2025-XX-XX
 --------------------
 
+**Changes**
+
+- Allow mixed sample and non-sample nodes to be output by ``write_vcf``.
```

Comment:
This is no longer grammatical, you've left a deleted clause in

Reply:
Whoops, fixed.